php71/Dockerfile

```diff
@@ -36,9 +36,9 @@ RUN curl -L -o /tmp/phpredis.tar.gz "https://github.com/phpredis/phpredis/archiv
     phpize && \
```

Comment:
Did you forget to add the pecl-yaml extension here?

Reply:
Yep. Don't know how. Should be good now.